### PR TITLE
Add final competition information email

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Add email chasing up MCFs. The original email was sent just over 2 weeks ago (#18), so chasing up around now feels about right.

### CODE REVIEW

File: SR2019/2019-03-27-final-competition-details.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Hi all,
+
+The competition is now less than 2 weeks away! We have some additional important information for you.
+
+Most importantly is media consent. Media consent forms must be sent to us by 5pm Thursday 4th April. After then most of our volunteers will be on site and not handling
+forms. If you've not returned a media consent form by then, it will take significantly longer to pass through reception, and some competitors may be turned away.
+
+You can download the media consent form from [our website](https://studentrobotics.org/events/sr2019/competition/).
+
+Parents and other visitors do not require tickets, but are required to sign media consent forms. These forms should be bought with them, or can be signed at reception.
```
Comment (PeterJCLaw) [inline SR2019/2019-03-27-final-competition-details.md]: s/bought/brought/
(You've made this typo before, is your keyboard's `r` key ok?)

File: SR2019/2019-03-27-final-competition-details.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Please ensure that your team bring at least 1 laptop or appropriate device with them to the competition to access the docs and IDE on. We are unable to provide any computing devices for competitor use at the competition. We will be providing a wireless internet connection to all teams.
+
+Any tools that your team requires to modify their robot must be brought to the competition by the team: we cannot provide any. In particular, the use of any dangerous tools must be supervised by the responsible adult for the team.
+
+All robots are subject to a safety check before the start of the competition (see appendix B in the rulebook). We may disqualifying any dangerous robots, so please ensure the team have checked their robot against the safety checklist.
```
Comment (PeterJCLaw) [inline SR2019/2019-03-27-final-competition-details.md]: > We may disqualifying any dangerous robots

Missing word here? I'd also upgrade "may" to something stronger -- perhaps "Robots which do not pass safety checks will not be allowed in the arena"?

File: SR2019/2019-03-27-final-competition-details.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+You must bring all of the robotics kit (including the white RUB it came in) to the competition, so that we can collect it at the end of the event, unless you've already arranged to keep the kit longer.
```
Comment (PeterJCLaw) [inline SR2019/2019-03-27-final-competition-details.md]: Will teams know what a "RUB" is? Perhaps either expand this or just say "plastic box"?

File: SR2019/2019-03-27-final-competition-details.md (hunk 1 above)
Comment (RealOrangeOne) [reply SR2019/2019-03-27-final-competition-details.md]: https://github.com/srobo/team-emails/pull/19/commits/194acd352a6742f9c5e64c553398619cd29a5d98

File: SR2019/2019-03-27-final-competition-details.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+Teams will be allocated an area of the venue to work in, and a table to work on. You will have plenty of chance to test your robot in the full arena during _Tinker Time_.
+
+## Safety
+
+All robots are subject to a safety check before the start of the competition (see appendix B in the rulebook). Robots which do not pass safety checks will not be allowed in the arena, so please ensure the team have checked their robot against the safety checklist.
+
+## Flag mounts
+
+All robots must be fitted with a flag mount to fit their corners flag. The flags themselves will be provided by Student Robotics. The specifications for the flags is available in section 3.2 of the rules.
```
Comment (PeterJCLaw) [inline SR2019/2019-03-27-final-competition-details.md]: Perhaps be explicit that this is a reminder?
```suggestion
As a reminder, all robots must be fitted with a flag mount to fit their corners flag. The flags themselves will be provided by Student Robotics. The specifications for the flags is available in section 3.2 of the rules.
```

File: SR2019/2019-03-27-final-competition-details.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+Teams will be allocated an area of the venue to work in, and a table to work on. You will have plenty of chance to test your robot in the full arena during _Tinker Time_.
+
+## Safety
+
+All robots are subject to a safety check before the start of the competition (see appendix B in the rulebook). Robots which do not pass safety checks will not be allowed in the arena, so please ensure the team have checked their robot against the safety checklist.
+
+## Flag mounts
+
+As a reminder, all robots must be fitted with a flag mount to fit their corners flag. The flags themselves will be provided by Student Robotics. The specifications for the flags is available in section 3.2 of the rules.
```
Comment (PeterJCLaw) [inline SR2019/2019-03-27-final-competition-details.md]:
```suggestion
As a reminder, all robots must be fitted with a flag mount to fit their corner's flag. The flags themselves will be provided by Student Robotics. The specifications for the flags is available in section 3.2 of the rules.
```

**Comment by RealOrangeOne** [comment]
This has been scheduled for 8am tomorrow.

**Pull request merged**